packages/web3-utils/src/converters.ts

```diff
@@ -70,36 +72,29 @@ export const ethUnitMap = {
 
 export type EtherUnits = keyof typeof ethUnitMap;
 /**
- * Convert a value from bytes to Buffer
+ * Convert a value from bytes to Uint8Array
  * @param data - Data to be converted
- * @returns - The Buffer representation of the input data
+ * @returns - The Uint8Array representation of the input data
  *
  * @example
  * ```ts
  * console.log(web3.utils.bytesToBuffer(new Uint8Array([72, 12])));
  * > <Buffer 48 0c>
  * ```
  */
-export const bytesToBuffer = (data: Bytes): Buffer | never => {
+export const bytesToBuffer = (data: Bytes): Uint8Array | never => {
```

Comment:
```suggestion
export const bytesToUint8Array = (data: Bytes): Uint8Array | never => {
```
should it be renamed for more accurate info, as it will convert string or hex string to uint 8 array https://github.com/web3/web3.js/blob/2258b2cbb4a7c9e15cbe6dbd07d29f10e4da9350/packages/web3-types/src/primitives_types.ts#L19

Reply:
updated